lib/src/base/_stacks-body.less

```diff
@@ -25,6 +25,13 @@ body {
     line-height: @lh-base;
 }
 
+#stacks-internals #screen-sm({
```

Comment:
@balpha This approach isn't working in the production codebase. I was able to drop in the prior approach. Do you mind poking at `feature/responsive-typography` in production?

Reply:
@aaronshekey Will do. Can you define "not working"?

Reply:
So it, well "works" here -- it's just that the body font size is overridden by another CSS rule later. Is that what you're referring to?

Reply:
If you comment out line 88 in `_base-primary.less`, it behaves as expected.

Reply:
Yep, that must be what's happening here. Thanks for tracking that down. I assumed it was a mixin issue and wasn't looking beyond that. :v: